README.md

```diff
@@ -1,31 +1,156 @@
 
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/). 
-Before running the tests make sure you are serving the app via `ng serve`.
+  * `git clone https://github.com/Sfeir/angular-200.git`
+  * `npm install -g @angular/cli` 
+  * `cd angular-200`
+  * `npm install`
 
-## Deploying to Github Pages
 
-Run `ng github-pages:deploy` to deploy to Github Pages.
+  * `npm run server`
 
-## Further help
 
-To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+  
+
+## Déroulement de la formation
+Un TP :
+
+  * un projet par exercice :
+`npm run client step01`
+  * un projet par solution :
+`npm run client step01solution`
+
+ 
+## Planning de la formation
+
+### Jour 1
+
+#### Mise en place d'Angular
+* Installation
+* La notion de composant
+* Bootstrap d'une application
+
+#### Le databinding
+* Variable
+* Propriétés
+* Evénements
+
+#### Les components
+* Annotations
+* Classes
+* Communication entre composants
+
+#### Le templating avancé
+* NgFor
+* Gestion des événements DOM
+
+#### Les datas
+* HTTP
+* Observables
+
+#### Le système de routage
+* Mise en place du routeur
+* Configuration
+* Directives
+
+
+### Jour 2
+
+#### Les formulaires et la validation
+* Différents éléments de formulaires
+* Template-driven versus Model-driven
+* Créer ses validators
+
+#### Les services
+* Créer un service
+* Utiliser un service
+
+#### L'injection de dépendances
+* Principe
+* Créer et utiliser des dépendances
+
+#### Les Pipes
+* Les pipes natifs
+* Créer son propre pipe
+
+#### Les directives
+* Les directives natives
+* Créer sa propre directive
+
+#### Initiation aux tests
+* La mise en place
+* Tests unitaires pour un service, un pipe et un component
+
+#### Initiation communication avancée
+* Bus d'événements
+* NGRX / NGXS
+
+
+
+
+## Liste des projets
+
+  * step01: Création d'un composant manuellement
+  * step02: Création d'un composant avec le CLI
+  * step03: Imbriquer les composants
+  * step04: Afficher une personne
+  * step05: Gérer un clic
+  * step06: Récupérer les personnes depuis le serveur
+  * step07: Configurer la navigation
+  * step08: Utiliser la directive *ngFor
+  * step09: Réutilisation des composants (partie 1)
+  * step10: Réutilisation des composants (partie 2)
+  * step11: Supprimer une personne
+  * step12: Ajouter une personne
+  * step13: Modifier une personne
+  * step14: Valider le formulaire (Template driven form)
+  * step15: Valider le formulaire (Model driven form)
+  * step16: Créer un validateur de formulaire
+  * step17: Créer un service
+  * step18: Utiliser une Pipe
+  * step19: Créer une Pipe
+  * step20: Utilisation des directives
+  * step21: Créer une directive
+  * step22: TDD
+  * step23: NGXS
+  * step24: NGRX
+
+
+
+## Stack technique
+
+* Angular
+* NGXS
+* NGRX
+* Angular Material
+* NodeJS
+* ExpressJS
+
+## Commandes utiles
```

Comment:
this is the most important part of the readme. Please replace ng by `npm run ng --` and add a short mention that we can use ng if we install the cli globally. Thx